JSIL/JavascriptAstEmitter.cs

```diff
@@ -78,6 +80,16 @@ Configuration configuration
             this.Visit(node);
         }
 
+        public override void Visit(JSNode node, string name = null)
+        {
+            base.Visit(node, name);
+            if (Output.SourceMapBuilder != null && node != null && node.SymbolInfo != null && node.SymbolInfo.Any())
+            {
+                Output.SourceMapBuilder.AddInfo(Output.Output.Line, Output.Output.FirstNonSpace, node.SymbolInfo);
```

Comment:
Really here should be `Output.Output.Column` instead of `Output.Output.FirstNonSpace`. But VS debugger doesn't work if there is no symbol for first non-space symbol on line.
While we still has only one point per line (while we take info from PDB), we could use this hack. We need some better solution in general.